server/src/controllers/user.ts

```diff
@@ -0,0 +1,44 @@
+import { Request, Response } from "express";
+import userService from "../services/user";
+
+const get = async (req: Request, resp: Response) => {
+  try {
+    const id: number = req.body.id;
```

Comment:
A get request should not have a `body` in its payload. Are you certain this works?

But even if it is, a RESTful API should have the `id` as part of the URL. So in the routes file the URLs should be `/:id` instead of `/`.

Reply:
This also applies to all of the PRs I have looked at thus far.

Reply:
Am aware of that but was in a hurry and tired, my bad. I tested everything with Postman and it works. Fixed 👍